File: backend/api/src/main/java/com/locuspark/api/service/report/export/format/VehicleTypeLabel.java
package com.locuspark.api.service.report.export.format;

import com.locuspark.api.enums.VehicleType;

public final class VehicleTypeLabel {

    private VehicleTypeLabel() {
    }

    public static String label(VehicleType type) {
        if (type == null) {
            return "—";
        }
        return switch (type) {
            case CAR -> "Carro";
            case MOTORCYCLE -> "Moto";
            case VAN -> "Van";
            case TRUCK -> "Caminhão";
        };
    }
}

File: frontend/src/app/shared/utils/vehicle-type-label.ts
import { VehicleType } from '../../core/types/domain-enums.types';

export const VEHICLE_TYPE_LABELS: Record<VehicleType, string> = {
  CAR: 'Carro',
  MOTORCYCLE: 'Moto',
  VAN: 'Van',
  TRUCK: 'Caminhão',
};

export function vehicleTypeLabel(type: VehicleType): string {
  return VEHICLE_TYPE_LABELS[type];
}
